content/minfraud/api-documentation/responses.mdx

```diff
@@ -0,0 +1,229 @@
+---
+draft: false
+title: minFraud API Responses
+---
+
+import * as responseJson from './_examples/response'
+import * as Schemas from './_schemas/';
+
+## Headers
+
+<table>
+  <thead>
+    <tr>
+      <th>Header</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        `Content-Type`
+      </td>
+      <td>
+        The `Content-Type` header for a successful response varies based on the
+        service as outlined above.
+
+        Errors may be returned with the `Content-Type` set to
+        `application/vnd.maxmind.com-error+json; charset=UTF-8; version=2.0`.
+        If this is the case, then the body of the response contains a JSON
+        document with two keys, code and error. See the [Errors](#) section for
+        more details.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `Content-Length`
+      </td>
+      <td>
+        The response will always include a `Content-Length` header as well.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Bodies
+
+Each service returns data as a JSON document. The document that is returned
+always consists of an object (aka map or hash). Below are the schema definitions
+that describe each service's response body.
+
+### Score Body
```

Comment:
The order of the objects is a little different than our current site (see screenshot). Is that expected? 
![image](https://user-images.githubusercontent.com/47989320/110953156-09638100-8315-11eb-8562-285ffdd9f3dc.png)

Reply:
Kevin and I talked this over. The code linter automatically alphasorted the properties, which is something we don't mind. It makes the object more scannable when you're having to read through it to find a specific property. This can be beneficial for both maintainers and users of the docs.